sources/packages/api/src/route-controllers/institution/institution.controller.ts

```diff
@@ -33,4 +36,17 @@ export class InstitutionController extends BaseController {
     // Save institution
     await this.institutionService.createInstitution(userToken, payload);
   } //create method ends
+
+  @Patch()
+  async update(
+    @Body() payload: InstituteDto,
+    @UserToken() userToken: IUserToken,
+  ) {
+    await this.institutionService.updateInstitution(userToken, payload);
+  }
+
+  @Get()
+  async institutionDetail(@UserToken() token: IUserToken) {
```

Comment:
Can we try to add the explicit return types always as possible? I think that we should take it as a pattern. Please let me know your thoughts on this.

Reply:
DONE. I missed that part if any method is not returning that mean either void or Promise<void> else it should return proper type or any